tests/torchtune/config/test_utils.py

```diff
@@ -9,9 +9,11 @@
 import pytest
 from torchtune.config._utils import (
     _get_component_from_path,
+    _get_template,
```

Comment:
Not the most pressing for this particular PR, but I find it quite confusing that we have separate `test_utils.py` files in our repo that either provide utilities for tests or test our utilities. We should clean this up

Reply:
maybe rename this to `test_config_utils`?